.all-contributorsrc

```diff
@@ -7,7 +7,7 @@
     "README.md"
   ],
   "imageSize": 100,
-  "commit": true,
+  "commit": false,
```

Comment:
Revert please, I still do it using CLI

Reply:
Changing this to false just doesn't run `git commit -am "Add...`. You can still commit manually.

Reply:
No. I rather just squash them instead of scrolling again who's being added.

I'll revert this since you won't touch CLI in the first place.

Reply:
I mean I do sometimes use cli but ok